src/wasm/literal.cpp

```diff
@@ -60,6 +60,9 @@ Literal::Literal(const Literal& other) : type(other.type) {
     new (&gcData) std::shared_ptr<Literals>(other.gcData);
   } else if (type.isFunction()) {
     func = other.func;
+  } else if (type.isRtt()) {
+    // Nothing to do: Rtts help JITs optimize, but are not used in the
+    // interpreter yet, and they are opaque to the wasm itself.
```

Comment:
Doesn't this PR add the interpreter support too?

Reply:
I didn't plan to, but the fuzzer did actually run into this, so I added it. But there is a lot more work necessary I think, for later.